README.md

```diff
@@ -1,4 +1,27 @@
+## Introduction
 
+**What?** The [Neurodata Without Borders: Neurophysiology][nwb-neurophysiology] (NWB:N) Hackathon is a week-long hands
+on activity in which neurophysiology researchers create solutions using the open source NWB:N software packages.
+Participants work collaboratively on solutions to develop a unified data format for cellular-based neurophysiology data, focused on the dynamics of groups of neurons measured under a large range of experimental conditions. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Hackathon is to provide a venue for creators of neurophysiology open-source software to collaboratively work.
+
+[nwb-neurophysiology]: http://www.nwb.org/nwb-neurophysiology/
+
+
+**When, where, how much?** Twice a year - Early April in Seattle, and late April in Berkley. Ad-hoc meetings are added occasionally. There is no registration fee, and lunch and dinner are usually covered.
+
+**How does it work?** Weekly videoconferences for preparation begin 8-12 weeks before an event. Potential participants propose projects during these meetings, and collaboratively create a list of projects that are of mutual interest. The projects include platform work, algorithm development, and neurophysiology applications. Through the course of the meetings, each participant selects one or more project teams and develops goals for the week. The first day of the Project Week itself begins with a 2-hour in-person introduction to all projects and participants. The rest of the week consists of a mix of working sessions and breakout sessions on special topics, as decided by the participants during the preparatory meetings.
+
+**Who can attend?** Hackathons are open to XXXX and publicly advertised. One-day participation is permitted for first-time attendees, but rest stay for the entire event. Email announcements are sent to the [Hackathon mailing list](To be done) (for continuity) and posted on the [Hackathon forum](https://github.com/orgs/NeurodataWithoutBorders/teams/hck01/discussions).
+
+**Who should attend?** This is harder to determine. Your best bet is to send an email or attend a preparatory videoconference (see below).
+
+**What else?**  The Hackathon series was founded in 2018 by the Allen Brain Institute to develop a unified data format for cellular-based neurophysiology data. The NWB:N team consists of neuroscientists and software developers who recognize that adoption of a unified data format is an important step toward breaking down the barriers to data sharing in neuroscience.
```

Comment:
The Hackathon series was founded in 2018 by the Allen Brain Institute to develop a unified data format for cellular-based neurophysiology data.

=>
The Hackathon series was founded in 2018 by the Allen Institute for Brain Science, Lawrence Berkeley National Lab, and Kitware to encourage the development, growth and use of NWB:N, a unified data format for cellular-based neurophysiology data.

Reply:
I'm not sure about the ``What else`` section. There have been at least 3 prior "hackathons" on creating NWB:N. I'm just concerned about giving the wrong impression that we started NWB in 2018. I would suggest to either a) add some sentence on the history of NWB or b) simply remove the "What else" part.

Reply:
This is great. Do you have link for each one of them ?  (Google doc, email, wiki, ...)

That way, we can start the numbering accordingly and add reference to the previous one ?

Reply:
"Allen Brain Institute" => "Allen Institute for Brain Science"

Reply:
I think @cmartin-kavlifoundation probably has more details on the NWB:N hackathons (or maybe there is already something about the history of NWB:N on nwb.org). Here my attempt at a rewrite of the "What else" section:

**What is the history of NWB hackathons?** The NWB:N team consists of neuroscientists and software developers who recognize that creation and adoption of a unified data format is an important step toward breaking down the barriers to data sharing in neuroscience. Hackathons are a way for us to collaborate and develop NWB:N as well as to engage with the NWB:N user community.  For an overview of NeurodataWithoutBorders see http://nwb.org/ . 

As part of the development of NWB:N 1.x two hackathons were held at Janelia Farm, in Ashburn, Virginia; the first hackathon  on November 20 – 22, 2014 and the second one on May 14-16 at, 2015 (http://crcns.org/NWB/). As part of the development of NWB:N 2.0 a first hackathon was held at Janelia Farm, in Ashburn, Virginia  on July 31 - August 1, 2017. The primary focus of the first three hackathons has been on development of the NWB:N format as well as on development of a software strategy for NWB:N. 

To encourage the development, growth and use of NWB:N as a unified data format for cellular-based neurophysiology data, the Allen Institute for Brain Science, Lawrence Berkeley National Lab, and Kitware are organizing a development-focused hackathon at the Allen Institute for Brain Science (April 3-6, 2018) and a user-engagement and training hackathon at Lawrence Berkeley National Laboratory (April 25-27).